notebooks/Model.ipynb

```diff
@@ -1,19 +1,19 @@
-{
+{MLOPSMLOPS
```

Comment:
remove this line,  I couldn't open the notebook because of it

Reply:
https://github.com/datarisk-io/mlops_codex/pull/64/commits/4eb4cd115b8408e34c538664bf9b9f25a99eb714